nixos/modules/services/misc/forgejo.nix

```diff
@@ -322,6 +322,20 @@ in
                 description = "Disable external SSH feature.";
               };
 
+              START_SSH_SERVER = mkOption {
+                type = types.bool;
+                default = false;
```

Comment:
This should be `mkEnableOption` since it's a simple default false boolean.

Reply:
Updated

Reply:
Last force-push/change broke eval.

The call signature of `mkOption` is different from `mkEnableOption`.

If you need help with that, let me know :)

Reply:
Yup, forgot to run a test build of the manual. Just resolved that error.